### PR TITLE
"No matching template" error message (RHV)

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1868868

I also moved the wizard fields closer to the wizard import procedures. This was not in the scope of the bug.

CP 4.5, 4.6

### CODE REVIEW

File: modules/virt-troubleshooting-vm-import.adoc [hunk 1]
```diff
@@ -87,6 +87,22 @@ Failed to bind volumes: provisioning failed for PVC
...
+----
+The virtual machine could not be imported.
+VMTemplateMatchingFailed: Couldn't find matching template
+----
++
+Check the following:
+
+** The guest agent correctly reports the operating system of the source VM.
+** The source VM operating system has a matching {VirtProductName} template.
+** The `vm-import-controller` ConfigMap maps the source VM operating system to the {VirtProductName} template.
```
Comment (istein1) [inline modules/virt-troubleshooting-vm-import.adoc:103]: Not sure what is the intention here.
If we get this error it means the existing template mapping do not include the source VM OS type.

File: modules/virt-troubleshooting-vm-import.adoc [hunk 2]
```diff
@@ -87,6 +87,22 @@ Failed to bind volumes: provisioning failed for PVC
...
+
+ifeval::["{HCOVersion}" == "2.4.1"]
+* The following error message is displayed in the *Virtual Machines* tab of the *Virtualization* screen in the {product-title} console if there is no matching {VirtProductName} template:
++
+----
+The virtual machine could not be imported.
+VMTemplateMatchingFailed: Couldn't find matching template
+----
++
+Check the following:
```
Comment (istein1) [inline modules/virt-troubleshooting-vm-import.adoc:99]: If this error appear it means that there is no mapping to a common template.

Resolution:
Either set source VM another OS type, that exist in the template mapping (maybe direct here to the related section in this doc), 
(For example if source VM OS is "Other", and this is a Windows2016 VM - Need to set it's OS to such).
or map it to a CNV common template, using a custom template configMap,
or add first a common template, if there is no such, and then add a mapping for it.

Comment (apinnick) [reply modules/virt-troubleshooting-vm-import.adoc:99]: Was just thinking that we don't really need to put all this info here. I think that the sentence indicating the cause of the error (no matching OCP template) will be enough.

I can't direct the user to the ConfigMap section because we can't use cross-references in modules.

File: modules/virt-troubleshooting-vm-import.adoc [hunk 3]
```diff
@@ -80,6 +82,22 @@ Failed to bind volumes: provisioning failed for PVC
 ----
 +
 You must use the NFS storage class. Cinder storage is not supported. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857784[(*BZ#1857784*)]
+
+ifeval::["{HCOVersion}" == "2.4.1"]
```
Comment (aburdenthehand) [inline modules/virt-troubleshooting-vm-import.adoc:86]: virt-document-attributes currently has this set to 2.4.0 so this isn't included in the current build. 
Is this attribute going to be updated in a different PR closer to release?

Comment (apinnick) [reply modules/virt-troubleshooting-vm-import.adoc:86]: I certainly hope so. 2.4.1 is supposed to be released tomorrow. Isn't someone on your team responsible for updating the attributes with each release?
